src/multipart_.rs

```diff
@@ -197,6 +203,21 @@ impl Part {
         self.file_name = Some(filename.into());
         self
     }
+
+    /// Returns a reference to the map of additional header fields
+    pub fn header_fields(&self) -> &HeaderMap {
+        &self.hdr
+    }
+
+    /// Returns a mutable reference to the map of additional header fields
+    pub fn header_fields_mut(&mut self) -> &mut HeaderMap {
+        &mut self.hdr
+    }
+
+    /// Replaces the additional header fields
+    pub fn replace_header_fields(&mut self, hdr: HeaderMap) -> () {
```

Comment:
This method shouldn't be needed, as anyone can do the same with `std::mem::replace` combined with `headers_mut`.